test/SourceKit/CursorInfo/cursor_callargs.swift

```diff
@@ -17,13 +17,15 @@ c1.meth(0, passFloat: 0)
 // CHECK-CLASS: -----
 // CHECK-CLASS-NEXT: SECONDARY SYMBOLS END
 
-// RUN: %sourcekitd-test -req=cursor -pos=6:17 %s -- %s | %FileCheck -check-prefix=CHECK-INIT %s
-// RUN: %sourcekitd-test -req=cursor -pos=6:31 %s -- %s | %FileCheck -check-prefix=CHECK-INIT %s
-// CHECK-INIT: source.lang.swift.ref.function.constructor (3:3-3:37)
+// RUN: %sourcekitd-test -req=cursor -pos=6:17 %s -- %s | %FileCheck -check-prefix=CHECK-PAR-ONE %s
+// RUN: %sourcekitd-test -req=cursor -pos=6:31 %s -- %s | %FileCheck -check-prefix=CHECK-PAR-TWO %s
+// CHECK-PAR-ONE: source.lang.swift.ref.var.local (3:8-3:15)
```

Comment:
If the function has documentation for each parameter, it would be nice if we could return the parameter documentation here. But I’m not sure if that’s easily possible.